src/awx_plugins/credentials/azure_kv.py

```diff
@@ -71,20 +86,41 @@
     ],
     'required': [
         'url',
-        'client',
-        'secret',
-        'tenant',
         'secret_field',
     ],
 }
 
 
-def azure_keyvault_backend(**kwargs):
-    csc = ClientSecretCredential(
-        tenant_id=kwargs['tenant'],
-        client_id=kwargs['client'],
-        client_secret=kwargs['secret'],
-    )
+def azure_keyvault_backend(**kwargs: str) -> str:
```

Comment:
Invalid typing. `kwargs` here is a dict, it'll be difficult to type it. It's best to list the individual args instead.

Reply:
I've listed the individual args.